### DISCUSSION TITLE
Skew property on input

### DISCUSSION THREAD

**Discussion by Axel-AMGHAR** [tailwindlabs/tailwindcss#10344] [Help]
Hello, 

When i'm using skew property on input. The cursor on the input is not showing properly.
I want to make a codepen on this but can't find some with tailwindcss cdn version3.

There is tailwind a play with an input that work, and want who does not so you will better understand:
https://play.tailwindcss.com/Ljxzmv1lZr

Can you help me on the topic.

TY.

**Comment by wongjn** [comment]
Please consider providing a [Tailwind Play](https://play.tailwindcss.com) that reproduces the undesired behavior.

**Reply by 52461** [reply]
Hello, 
First, select the element where the cursor element needs to hide.
Add CSS style cursor: none to the class.
Add the class name (class name of CSS style cursor:none) to the particular element where the cursor element is to be hidden.   [MyGeorgiaSouthern](https://www.mygeorgiasouthern.org/)

**Reply by Axel-AMGHAR** [reply]
oc. I edited my original post.

**Comment by Axel-AMGHAR** [comment]
It seems it is solved on v3 and above.
